src/main/java/com/trailfinder/service/HikeServiceStub.java

```diff
@@ -23,31 +23,14 @@
 	public ProfileDTO getProfile(int profileId) {
 		// TODO create a List of profiles and check if the passed Id matches any
 		List<ProfileDTO> profiles = new ArrayList<ProfileDTO>();
-		// Create 5 profiles
-		int i;
-		for (i = 0; i < 5; i++)
-		{
-			ProfileDTO testProfile = new ProfileDTO();
-			testProfile.setProfileId(i);
-			testProfile.setFirstName("TestName" + i);
-			testProfile.setBio("TestBio");
-			testProfile.setEvents(null);
-			profiles.add(testProfile);
-		}
 		
-		// Check which profile was selected
-		ProfileDTO profile = new ProfileDTO();
-		switch (profileId) 
+		// Create 5 profiles
+		for (int i = 0; i < 5; i++)
```

Comment:
Changed the scope of i since it was only being used in the for loop.

Reply:
That makes sense, yes.